src/documents/mail.py

```diff
@@ -75,8 +75,9 @@ def __init__(self, data, group=None):
                 continue
 
             dispositions = content_disposition.strip().split(";")
-            if not dispositions[0].lower() == "attachment" and \
-               "filename" not in dispositions[1].lower():
+            if len(dispositions) < 2 or \
```

Comment:
3-line if statements make me cringe a little.  I didn't much like it when it was 2-lines, but at 3 we're pushing it.  Can you maybe break this up into multiple if statements just for readability?

Reply:
Done.